cinder/nav.html

```diff
@@ -42,11 +42,13 @@
             {% endif %}
 
             <ul class="nav navbar-nav navbar-right">
-                <li>
-                    <a href="#" data-toggle="modal" data-target="#mkdocs_search_modal">
-                        <i class="fa fa-search"></i> Search
-                    </a>
-                </li>
+                {% if include_search %}
```

Comment:
Has this been tested? Looking at the MkDocs documentation, it should be `if config.extra.include_search`.